www/layouts/resources/single.html

```diff
@@ -4,8 +4,8 @@
 {{ end }}
 <div class="container mt-5">
   <div class="rounded mb-2">
-    {{ partial "content_header.html" . }}
+    {{ partial "content_header_v2.html" . }}
```

Comment:
I was curious about this change since it's the only change that could conceivably affect the production site.

And it does. But I think that's OK:
- The change seems desirable: we should have done this when v2 is released. Now www resources look more similar to course resources

Before:
<img width="1287" alt="Screenshot 2023-01-25 at 7 44 28 AM" src="https://user-images.githubusercontent.com/9010790/214567746-beed16b0-ed07-48f7-9468-aee15b3222d7.png">

Now:
<img width="1150" alt="Screenshot 2023-01-25 at 7 44 21 AM" src="https://user-images.githubusercontent.com/9010790/214567791-ec645acc-aaf4-4011-9714-c1583fd6d129.png">

Reply:
> The change seems desirable: we should have done this when v2 is released. Now www resources look more similar to course resources

I agree!